modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -3276,6 +3277,36 @@ export class Wallet implements IWallet {
     }
   }
 
+  /**
+   * Sends a transaction from a TSS (Threshold Signature Scheme) wallet.
+   *
+   * @param params
+   *    txRequestId - The ID of the transaction request.
+   *    walletPassphrase - The passphrase for the wallet.
+   *    isTxRequestFull - Flag indicating if the transaction request is full.
+   * @returns A promise that resolves to a SignedTransaction.
+   */
+  public async sendHotWalletAllTSSWithdrawalTransaction(
```

Comment:
```suggestion
  public async signAndSendTxRequest(
```

Reply:
Also improve the jsDoc to say what it does, it just signs and send a txRequest.
Since is not really building the transaction, send withdrawal transaction doesn't make sense.
Also technically also works for cold wallets (SMC) if they are using external signer.